source/includes/_registry.md

```diff
@@ -0,0 +1,106 @@
+# Registry
+
+The registry is the low-level host functionality that provides the foundation
+for [SkyDB](#skydb). It allows getting and setting registry entries with a
+user's key and a data key (e.g. the name of an app).
```

Comment:
Let's not mention skydb from registry docs, it's the other way around that we should mention.